poster/styles/styles.scss

```diff
@@ -11,480 +7,451 @@ sans-serif;
-    line-height: 2.2vw;
-}
+    &nineteen {
+        margin-left: 6vw;
+    }
 
-.firstcoloumn {
-    grid-column: 1/4;
-    grid-row: 1/2;
-}
+    &twenty {
+        margin-left: 12vw;
+    }
 
-/*header in top half*/
-h1 {
-    font-size: 24vw;
-    margin-top: -10vw;
-    margin-bottom: 0;
-    display: grid;
-    grid-template-columns: repeat(7, 14vw);
-    grid-template-rows: repeat(20, 4vw);
-}
+    &twentyone {
+        margin-left: 16vw;
+    }
 
-h2 {
-    font-size: 2.6vw;
-    font-weight: bold;
-}
+    &twentytwo {
+        margin-top: 10vw;
+        margin-left: 20vw;
+    }
 
-h3 {
-    font-size: $font-size;
-    font-weight: $font-weight;
-}
+    &twentythree {
+        margin-left: 16vw;
+    }
 
-li {
-    font-size: $font-size;
-    font-weight: $font-weight;
-}
+    &twentyfour {
+        margin-top: -1vw;
+        margin-left: 14vw;
+        margin-bottom: -1.5vw;
+    }
 
-/*container for whole compostion seperating in top and bottom halves*/
+    &twentyfive {
+        margin-left: 7vw;
+    }
 
-.outerwrapper {
-    display: grid;
-    grid-template-columns: 90%;
-    grid-template-rows: 1fr 1fr;
-    grid-gap: 0;
-    padding-left: 0;
-    color: #FAFAF2;
-}
+    &twentysix {
+        margin-left: 10vw;
+    }
 
-p {
-    font-size: $font-size;
-    font-weight: $font-weight;
-}
+    &twentyseven {
+        margin-left: 8vw;
+    }
 
-.secondcoloumn {
-    grid-column: 4/8;
-    grid-row: 1/21;
-}
+    &twentyeight {
+        margin-left: 6vw;
+    }
 
-/* title letters- different fonts and locations */
-.titlefirstletter {
-    font-family: $abstract-display;
-    grid-column: 1/2;
-    grid-row: 12/21;
-}
+    &twentynine {
+        margin-left: 4vw;
+    }
 
-.titlesecondletter {
-    font-family: $abstract-display;
-    grid-column: 2/3;
-    grid-row: 15/21;
-}
+    &thirty {
+        margin-left: 16vw;
+        margin-bottom: -1.5vw;
+    }
 
-.titlethirdletter {
-    font-family: $thin-display;
-    grid-column: 3/4;
-    grid-row: 12/16;
-}
+    &thirtyone {
+        margin-left: 8vw;
+    }
 
-.titlefourthletter {
-    font-family: $thin-display;
-    grid-column: 4/5;
-    grid-row: 14/20;
-}
+    &thirtytwo {
+        margin-left: 4vw;
+    }
 
-.titlefifthletter {
-    font-family: $thick-display;
-    font-size: 28vw;
-    grid-column: 5/6;
-    grid-row: 1/7;
-    margin-left: -6vw;
-    transform: rotate(-30deg);
-}
+    &thirtytwosecond {
+        margin-left: 2vw;
+    }
 
-.titlesixthletter {
-    font-family: $thin-display;
-    grid-column: 6/7;
-    grid-row: 14/20;
-}
+    &thirtythree {
+        margin-left: 8vw;
+    }
 
-.titleseventhletter {
-    font-family: $thin-display;
-    grid-column: 7/8;
-    grid-row: 10/16;
-}
+    &thirtyfour {
+        display: inline-block;
+        margin-top: 2vw;
+        margin-left: 18vw;
+    }
 
-/*title location within outer wrapper*/
+    &thirtyfive {
+        margin-left: 16vw;
+    }
 
-.tophalf {
-    grid-row: 1/2;
-    grid-column: span;
```

Comment:
This rule is broken. The `span` keyword only works with a value -- e.g., `span 3` to mean "this should span the next 3 columns."